camera_calibration/src/camera_calibration/calibrator.py

```diff
@@ -637,11 +637,14 @@ def cal_fromcorners(self, good):
         # If FIX_ASPECT_RATIO flag set, enforce focal lengths have 1/1 ratio
         self.intrinsics[0,0] = 1.0
         self.intrinsics[1,1] = 1.0
-        cv2.calibrateCamera(
+        reproj_err, self.intrinsics, self.distortion, rvecs, tvecs = cv2.calibrateCamera(
                    opts, ipts,
                    self.size, self.intrinsics,
                    self.distortion,
                    flags = self.calib_flags)
+        # OpenCV returns > 8 coefficients (all zeros) when CALIB_RATIONAL_MODEL is set.
+        # These include, e.g. thin prism coefficients, which we are not interested in.
```

Comment:
I see a self.distortion as input, output, and then remapped to [:8]. Lets clean this up to make it clear whats being stored or deleted. Only 1 thing should be stored as self.distortion.

Reply:
I updated the code to avoid using `self.intrinsics` and `self.distortion` as inputs for clarity.